rotkehlchen/exchanges/binance.py

```diff
@@ -847,6 +849,27 @@ def query_online_trade_history(
 
             raw_data.sort(key=lambda x: x['time'])
 
+        # Could also utilize start_ts and end_ts in request
```

Comment:
you have left commented out code here.

Reply:
this was for the fiat/payments endpoint, was not sure how to integrate the calls into query_online_trade_history